library-aux/src/scala/andType.scala

```diff
@@ -0,0 +1,7 @@
+package scala
+
+/** The intersection of two types.
+ *
+ *   See [[https://docs.scala-lang.org/scala3/reference/new-types/intersection-types.html]].
+ */
+type &[A, B] = A & B
```

Comment:
This awfully looks like a circular definition. Consider leaving it abstract instead?
```suggestion
type &[A, B]
```
Same for `|`.